Changelog.md

```diff
@@ -3,6 +3,112 @@
   mdformat --wrap 80 --end-of-line lf Changelog.md
 -->
 
+<!--
+## CLI
+## Steps
+## Flows
+## Tool Updates
+## Testing
+## Misc. Enhancements/Bugfixes
+## API Breaks
+## Documentation
+-->
+
+# 2.0.0rc1
+
+## CLI
+
+* Fixed `--ef-save-views-to` not saving `.mag`, `.gds` views
+
+## Steps
+
+* `Checker.*`
+  * Created new `ERROR_ON` variables with the removed `QUIT_ON` variables from
+    the `Classic` flow being added as deprecated names for these variables.
+  * Disabling `ERROR_ON*` (previously `QUIT_ON*`) no longer bypasses a step
+    entirely. A warning will be generated and the flow will not quit.
+* `Checker.TimingViolations`
+  * `TIMING_VIOLATIONS_CORNERS` renamed to `TIMING_VIOLATION_CORNERS`
+  * Now creates a new variable `<violation_type>_VIOLATION_CORNERS` for its
+    subclasses. This allows for fine-grained control of the IPVT corners checked
+    by each subclass.
+  * Now generates errors for violations occuring at `TIMING_VIOLATION_CORNERS`
+    (unless the subclass variable override is defined) and generates warnings
+    for violations in the rest of the IPVT corners.
+* `Checker.HoldViolations`
+  * Added `HOLD_VIOLATION_CORNERS` which takes precedence over
+    `TIMING_VIOLATION_CORNERS`
+* `Checker.SetupViolations`
+  * Added `SETUP_VIOLATION_CORNERS` acting similar to `HOLD_VIOLATION_CORNERS`.
+* Created `Checker.MaxCapViolations`
+  * Reports maximum capacitance violations.
+  * Added `MAX_CAP_VIOLATION_CORNERS`. It defaults to `[""]` which is the value
+    for matching no corners (i.e. all violations are reported as warnings).
+* Created `Checker.MaxSlewViolations`
+  * Reports maximum slew violations.
+  * Added `MAX_SLEW_VIOLATION_CORNERS`. It defaults to `[""]` which is the value
+    for matching no corners (i.e. all violations are reported as warnings).
+* `OpenROAD.GlobalPlacementSkipIO`
+  * Fixed a bug where `PL_TARGET_DENSITY_PCT` is calculated base on
+    `FP_CORE_UTIL` in designs with `FP_SIZING` set to `absolute`. The behavior
+    now matches `OpenROAD.GlobalPlacement`.
+* `Yosys.*`
+  * Verilog files are now read with `-noautowire`. This changes the default
+    `default_nettype` to `none`, no longer tolerating implicitly declared wires.
```

Comment:
maybe highlight that this doesn't override the value set by the input verilog file

Reply:
It feels fairly obvious?

Reply:
I updated it anyway